ow_faults_detection/src/FaultDetector.cpp

```diff
@@ -20,18 +20,33 @@ constexpr std::bitset<3> FaultDetector::isThermalError;
 FaultDetector::FaultDetector(ros::NodeHandle& node_handle)
 {
   srand (static_cast <unsigned> (time(0)));
-
- auto image_trigger_str = "/StereoCamera/left/image_trigger";
+  // antenna
+  auto original_str = "/_original";
+  auto ant_pan_str = "/ant_pan_position_controller";
+  auto ant_tilt_str = "/ant_tilt_position_controller";
+  m_ant_pan_command_sub = node_handle.subscribe( string("/_original") + ant_pan_str + string("/command"),
```

Comment:
I don't expect the `/_original/**` topics to be referenced in `FaultDetector.cpp`